yarn.lock

```diff
@@ -12450,4 +12450,4 @@ yocto-queue@^0.1.0:
 zksync-web3@^0.8.1:
   version "0.8.1"
   resolved "https://registry.npmjs.org/zksync-web3/-/zksync-web3-0.8.1.tgz#db289d8f6caf61f4d5ddc471fa3448d93208dc14"
-  integrity sha512-1A4aHPQ3MyuGjpv5X/8pVEN+MdZqMjfVmiweQSRjOlklXYu65wT9BGEOtCmMs5d3gIvLp4ssfTeuR5OCKOD2kw==
+  integrity sha512-1A4aHPQ3MyuGjpv5X/8pVEN+MdZqMjfVmiweQSRjOlklXYu65wT9BGEOtCmMs5d3gIvLp4ssfTeuR5OCKOD2kw==
```

Comment:
Can we remove this file from the diff? No changes except for the newline, might cause conflicts in future

Reply:
Done